frontend/src/metabase/querying/components/FilterPicker/FilterColumnPicker/FilterColumnPicker.tsx

```diff
@@ -110,6 +110,9 @@ export function FilterColumnPicker({
       renderItemName={renderItemName}
       renderItemDescription={omitItemDescription}
       renderItemIcon={renderItemIcon}
+      // disable scrollbars inside the list
+      style={{ overflow: "visible" }}
```

Comment:
why do you need `style`? Works in the same way for me with and without this line

Reply:
Otherwise it has `overflow: auto` inside, which is not expected